transport/tchannel/handler_test.go

```diff
@@ -663,3 +663,32 @@ func TestGetSystemError(t *testing.T) {
 		})
 	}
 }
+
+func TestIsTChannelTimeoutError(t *testing.T) {
+	tests := []struct {
+		giveErr  error
```

Comment:
let's add a `name string` property and use that in the `t.Run(tt.Name, ...` instead of using an integer

Reply:
Good point. Added